OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -467,7 +467,12 @@ Action InitInputPanel(Widget panel)
 				var hotkeys = new Dictionary<string, string>()
 				{
 					{ "ObserverCombinedView", "All Players" },
-					{ "ObserverWorldView", "Disable Shroud" }
+					{ "ObserverWorldView", "Disable Shroud" },
+					{ "PauseKey", "Pause/Play" },
```

Comment:
Really minor nitpick, you don't have to do this unless you feel like it: I'd put the pause key at the end of the list, so that the four speed modifier keys are next to each other in two lines, just for symmetry's sake.